samples/src/main/java/io/zeebe/example/cluster/TopologyViewer.java

```diff
@@ -11,16 +11,47 @@
 import io.zeebe.client.ZeebeClientBuilder;
 import io.zeebe.client.api.response.Topology;
 
+/**
+ * Sample application that connects to a cluster on Camunda Cloud, or a locally deployed cluster.
```

Comment:
Same suggestion as in https://github.com/zeebe-io/zeebe-get-started-java-client/pull/6